api.js

```diff
@@ -1371,15 +1436,42 @@ module.exports = (function () {
         .send({ error: 'BACKEND.ERROR.TYPE.PARAMS_REQUIRED' });
     }
     req.params.key = req.body.gid;
+    var type = req.body.type;
+    var g_id = req.body.gid;
     var successFn = ld.partial(function (req, res) {
       try {
+        console.log(req.body.gid);
+        group.get(g_id, function(err, g){
+          // var watchers = userCache.fn.getIdsFromLoginsOrEmails(g.watchers);
+          console.log(g.watchers);
+          for(var i=0; i< g.watchers.length; i++){
```

Comment:
Add a safety check in case the group does not have a watchers attribute (caused by legacy data)